bin/varnishncsa/varnishncsa.c

```diff
@@ -163,8 +163,42 @@ static struct ctx {
 	int			recv_compl;
 } CTX;
 
+enum format_policy {
+        FMTPOL_FIRST = 0,
+        FMTPOL_REQ,
+        FMTPOL_RESP,
+};
```

Comment:
My suggestion was to map the policy to the 3 categories "internal", "received" and "sent" and in our offline discussion I agreed to req/resp instead of sent/recd to make the change less intrusive than I suggested (and noted in https://github.com/varnishcache/varnish-cache/pull/4213#discussion_r1824229723).


`FMTPOL_FIRST` should really be `FMTPOL_INTERNAL` and there is no need to give it a specific value.